src/index.css

```diff
@@ -82,4 +83,130 @@ button {
 
 button:hover {
   background-color: orange;
-}
\ No newline at end of file
+}
+
+.navbar {
+  background-color: aqua;
+  margin: 1rem;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: orange;
+  padding: 0 24px;
+  box-shadow: rgba(0, 0, 0, 0.24) 0 3px 8px;
+}
+
```

Comment:
- [ ] Kindly use what you already learn in [React basic styling](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md) lesson for styling your app as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md#:~:text=In%20this%20project%2C%20you%20should%20use%20React%20basic%20styling.) `(`inline styles like [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md#:~:text=Method%201%3A%20Inline,CSS%20rules%3A), using different CSS files one for each component like [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md#:~:text=Method%202%3A%20Importing,way%20as%20usual%3A), or using CSS modules like [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md#:~:text=Method%203%3A%20CSS,module.css%20extension%3A) `)` instead of adding general style in the `index.css` file.

Reply:
Thank you @gemmen29 Fixing this